source/_patterns/components/secondary-navigation/secondary-nav.hbs

```diff
@@ -1,6 +1,6 @@
   <ul class="pf-secondary-nav" role="tablist">
     <li class="pf-secondary-nav__item">
-      <a href="#" role="tab" class="pf-secondary-nav__link" aria-selected="true">Secondary Tab One</a>
+      <a href="#" role="tab" class="pf-secondary-nav__link is-active" aria-selected="true">Secondary Tab One</a>
```

Comment:
So if I understand correctly, in a set of cards, the card with focus (your `is-active`) would be `aria-current`, and ones that are selected would be `aria-selected`. But if the two aria attributes mean the same for a widget (like a set of tabs), then you use `aria-selected`. Is the advantage that `is-active` would apply to both `aria-selected` tabs and `aria-current` cards? Is that confusing then with `aria-selected` cards?
I'm reading https://www.w3.org/TR/wai-aria-1.1/#aria-current
and https://www.w3.org/TR/wai-aria-practices/#tabpanelhttps://www.w3.org/TR/wai-aria-practices/#tabpanel
and https://www.w3.org/TR/wai-aria-practices/#TreeView (since it's a multiselect as cards could be)